homeassistant/components/fan/xiaomi_miio_services.yaml

```diff
@@ -31,6 +31,22 @@ xiaomi_miio_set_led_off:
       description: Name of the air purifier entity.
       example: 'fan.xiaomi_air_purifier'
 
+xiaomi_miio_set_child_lock_on:
```

Comment:
Service descriptions are no longer registered when registering the service but are now auto-discovered. To be auto-discovered, this file needs to be merged into `fan/services.yaml`.